mantidproject/mantid
# Sliceviewer: Capture arrow keys to move cursor while line plotting

**Description of work.**

For images with small pixels it can be difficult to move the mouse cursor to the required pixel. This work adds key handlers to process the keyboard arrow keys to move the cursor in the requested direction by a single pixel. 


**To test:**

* Load a workspace, e.g. `DocTest/MAR11015.raw`
* Show the sliceviewer and activate the lineplots
* Move the mouse and the lines should update as normal
* Leave the mouse over the image and use the arrow keys to move - the plots should update. 

Fixes #29235


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
